src/appMain/sdl_preloaded_pt.json

```diff
@@ -3403,7 +3407,44 @@
                     "name": "prndl",
                     "key": "OEM_REF_PRNDL",
                     "type": "PRNDL",
-                    "mandatory": false
+                    "mandatory": false,
```

Comment:
@IGapchuk move to a separate commit with comment that it is redundent

Reply:
Done in the https://github.com/smartdevicelink/sdl_core/pull/3401/commits/964eed04967e8a32b6a0109e93ce577f9666a420